### ISSUE TITLE
Exists doesn't work on custom wikibase instance

### ISSUE THREAD

**Issue by alemela** [maxlath/wikidata-edit#5]
Unfortunately a custom wikibase instance can't use all of your functions because of some hardwritten wikidata url on sdk.

I have found this problem using `exists`: https://github.com/maxlath/wikidata-sdk/blob/master/lib/utils/build_url.js#L1 I don't know if other functions are affected.

Thanks for trying to support also custom instances!

**Comment by maxlath** [comment]
Might be somewhat dirty, but for the moment, I recommend [customizing `wikidata-sdk` urls afterwards](https://github.com/maxlath/wikidata-sdk/blob/master/docs/use_a_custom_wikibase_instance.md).

[`wikidata-edit` requests are expected to do this customization](https://github.com/maxlath/wikidata-edit/blob/10425a24e2a930946581ef6c7791cbdbb761e460/lib/request.js#L18), so if it fails, it might be that an error in this process. You could try adding `console.log('url', url)` just after the [customization](https://github.com/maxlath/wikidata-edit/blob/10425a24e2a930946581ef6c7791cbdbb761e460/lib/request.js#L18) to see how that goes.

**Comment by alemela** [comment]
You were right! The url is ok, so is not a problem relating to customizing url. Using verbose mode in fact:
```
{ error: 
   { code: 'readapidenied',
     info: 'You need read permission to use this module',
     '*': 'See http://www.linkedcoins.org/wikibase/api.php for API usage' } }
```
Trying also from the browser: http://www.linkedcoins.org/wikibase/api.php?action=wbgetentities&ids=Q15&format=json send the same error. Maybe is a bad configuration from wikibase (maybe the default configuration...). Do you agree?

**Comment by maxlath** [comment]
The issue is probably that, based on wikidata.org wikibase instance behaviors, cookies weren't required to be passed to [GET requests](https://github.com/maxlath/wikidata-edit/blob/master/lib/request.js#L17-L21), at the difference of [POST requests](https://github.com/maxlath/wikidata-edit/blob/master/lib/request.js#L25-L26). That shouldn't be too complex to fix.

**Comment by maxlath** [comment]
@alemela could you try the newly published v1.3.1?

**Comment by alemela** [comment]
With 1.3.1 works like a charm!
